src/uct/api/v2/uct_v2.h

```diff
@@ -240,6 +251,18 @@ typedef enum {
 } uct_md_mem_dereg_flags_t;
 
 
+typedef enum {
+    /**
+     * Should be used in case if memory handle is supposed to be invalidated in
+     * any time, for example in the meddle of RMA operation to interrupt it.
+     * Such packed key is stronger from remote access perspective.
+     * @note The memory domain must support @ref UCT_MD_FLAG_INVALIDATE
```

Comment:
Indicates that remote access to the returned remote key should fail if uct_md_mem_dereg_v2 is called with the flag UCT_MD_MEM_DEREG_FLAG_INVALIDATE on the memory handle.
If this flag is not set, using the remote key after invalidating the memory region has an undefined behavior.